test/libsolidity/syntaxTests/metaTypes/runtimeCode_from_ternary_with_type_expression.sol

```diff
@@ -0,0 +1,24 @@
+contract A {
+    function f() public {}
+}
+
+contract B {
+    function g() public {}
+}
+
+contract C {
+    function ab(bool getA) pure public returns (bytes memory) {
+        return (getA ? type(A) : type(B)).runtimeCode;
+    }
+
+    function a() pure public returns (bytes memory) {
+        return type(A).runtimeCode;
+    }
+
+    function b() pure public returns (bytes memory) {
+        return type(B).runtimeCode;
+    }
```

Comment:
I think this one should already be disallowed by #14371.

Also, now that this is no longer a semantic test, `a()` and `b()` are superfluous. Same for `intMin()` and `intMax()` in `max_keyword_from_ternary_with_type_expression.sol`. In a semantic test they were there so that we can see that they return the correct value. In a failing syntax test they just make it harder to see which thing fails.